### ISSUE TITLE
atribut option --get-url not found

### ISSUE THREAD

**Issue by amidevous** [yt-dlp/yt-dlp#4131]
### Checklist

- [X] I'm requesting a feature unrelated to a specific site
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2022.05.18** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Description

attribute command line not identical to youtube-dl

so that we can update to your project it is very important that you reproduce all the options of youtube-dl identically

you can add new

but the old ones that are here must be identical and return the same answer in return

in my case youtube-dl and use in a php system encode in ioncube and which therefore cannot be modified or updated so impossible to replace as long as all the options present in youtube-dl are not present in yt-dlp

in particular the option -g or --get-url it is very used by the project that I have which can recover the links to work with php

both must be reproduced and return the same result

example with youtube-dl

`
youtube-dl -f 137 --get-url https://www.youtube.com/watch?v=4FkldLdZig0
https://rr4---sn-4g5edndz.googlevideo.com/videoplayback?expire=1655867599&ei=bzSyYqegJ4OK6dsPue-vuAM&ip=141.94.204.119&id=o-AGHinkWi18m2StB6TkVy3_buiCiQKV_-_nVBvuKqteW-&itag=137&aitags=133%2C134%2C135%2C136%2C137%2C160%2C242%2C243%2C244%2C247%2C248%2C271%2C278%2C313&source=youtube&requiressl=yes&mh=9_&mm=31%2C29&mn=sn-4g5edndz%2Csn-4g5lzne6&ms=au%2Crdu&mv=m&mvi=4&pl=22&initcwndbps=275000&spc=4ocVCz6HzB7JcBZbLrKcJbaqVW39ges&vprv=1&mime=video%2Fmp4&ns=YwZ4kDgNUM8RCIT2z9WjYxQG&gir=yes&clen=458657911&dur=880.533&lmt=1653400047749087&mt=1655845728&fvip=5&keepalive=yes&fexp=24001373%2C24007246&c=WEB&txp=4432434&n=cXMOitfdZ-xQymriSd&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cspc%2Cvprv%2Cmime%2Cns%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIgIGLfX2ypG_scQOJWBiVLvxnpLD_DLuUXc2LkhzhjA4UCIQCFBdh_sHwoBLRPdJpZu41x4eBtM6aHAcogW19DT5w6yg%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgPylHVER0cVdK8xDQZ7W2ZTGG7E0PDnCkiPb7CU0dAegCIQC7bBA5zBLM7FyFbsMNFyzXh8U2usr8KulLb0a_eN49nA%3D%3D
`

it is very important for compatibility and to allow everyone without any exception to be able to update

only the options that are present here in youtube-dl

https://github.com/ytdl-org/youtube-dl/blob/master/youtube_dl/options.py

be reproduced identically here in yt-dlp

https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/options.py

thank you

### Verbose log

```shell
youtube-dl --help | grep get-url
    -g, --get-url                        Simulate, quiet but print URL
root@vps-7ecc1d84:~# yt-dlp --help | grep get-url
root@vps-7ecc1d84:~#
```

**Comment by pukkandan** [comment]
`--get-url` is deprecated and thus no longer documented in `--help`. But the option still works, and there is no plan to remove it in the near future. See https://github.com/yt-dlp/yt-dlp#redundant-options

**Comment by amidevous** [comment]
well, no, it doesn't work and I don't see why it is obsolete

there are plenty of projects like the one I use php system encode in ioncube

for which we cannot modify the codes and which requires

so it must absolutely not be obsolete

there are many systems that need to have the url to work with directly

**Comment by amidevous** [comment]
yes but precisely its should not be obsolete it is a vital option especially for the project that I use it is a vital option

and considering it's in php encode with ioncube we absolutely can't modify the code to adapt it

it was you who decided that it should be obsolete

and those without thinking who has people for whom it is vital that the options remain the same

**Comment by pukkandan** [comment]
bro, like I said, the option still works! Unless you can show verbose log to the contrary, there is nothing to be done here
